### ISSUE TITLE
Docker error with BiomedParse v2

### ISSUE THREAD

**Issue by Mohinta2892** [microsoft/BiomedParse#84]
Hello @theodore-zhao ,

Wondering what I am doing incorrectly when I run `docker run biomedparse:latest`, because it throws an error expecting a `workspace/inputs/` folder.

```docker run biomedparse:latest 

==========
== CUDA ==
==========

CUDA Version 11.8.0

Container image Copyright (c) 2016-2023, NVIDIA CORPORATION & AFFILIATES. All rights reserved.

This container image and its contents are governed by the NVIDIA Deep Learning Container License.
By pulling and using the container, you accept the terms and conditions of this license:
https://developer.nvidia.com/ngc/nvidia-deep-learning-container-license

A copy of this license is made available in this container at /NGC-DL-CONTAINER-LICENSE for your convenience.

WARNING: The NVIDIA Driver was not detected.  GPU functionality will not be available.
   Use the NVIDIA Container Toolkit to start this container with GPU support; see
   https://docs.nvidia.com/datacenter/cloud-native/ .

==== Starting prediction ====
/workspace/inference.py:67: UserWarning: 
The version_base parameter is not specified.
Please specify a compatability version level, or None.
Will assume defaults for version 1.1
  hydra.initialize(config_path="configs", job_name="example_prediction")
/usr/local/lib/python3.10/dist-packages/hydra/_internal/defaults_list.py:251: UserWarning: In 'seem_model_3D': Defaults list is missing `_self_`. See https://hydra.cc/docs/1.2/upgrades/1.0_to_1.1/default_composition_order for more information
  warnings.warn(msg, UserWarning)
Using device: cpu
Checkpoint loaded successfully!
Traceback (most recent call last):
  File "/workspace/inference.py", line 134, in <module>
    main(args)
  File "/workspace/inference.py", line 76, in main
    for file in os.listdir(args.input_dir):
FileNotFoundError: [Errno 2] No such file or directory: '/workspace/inputs/'
```

Thank you!

**Comment by theodore-zhao** [comment]
Can you try the option to extract the model weights from the docker image? Let me know if that incurs any error
